src/test/java/org/takes/rq/form/RqFormFakeTest.java

```diff
@@ -76,6 +73,7 @@ public void createsFormRequestWithParams() throws Exception {
             key, avalue,
             akey, aavalue
         );
+        new RqPrint(req).printBody(System.err);
```

Comment:
@andreoss Why is it important to add this statement ?

Reply:
@baudoliver7 Removed (used it for debugging)